RELEASE_NOTES.md

```diff
@@ -1,5 +1,104 @@
 # Release Notes
 
+## Staging 2019/03/06
+
+### Ingest
+
+#### Broker
+
+Version: v0.8.8.rc
+
+* minor fix to code
+
+#### Core
+
+Version: v0.8.4.rc
+
+* authentication related security related patches
+
+### Upload Service
+
+Version: v4.2.3
+
+* Reraise original errors where tenacity retries are used instead of throwing a tenacity retry error
+* Bug fix for large file checksum notification flow
+* Reraise original errors where tenacity retries are used instead of throwing a tenacity retry error
```

Comment:
Accidentally put in duplicate line in the release notes. Could we delete one of them?

Reply:
Done!